mmpose/datasets/pipelines/top_down_transform.py

```diff
@@ -307,14 +316,112 @@ def _generate_target(self, cfg, joints_3d, joints_3d_visible):
 
         return target, target_weight
 
+    def _megvii_generate_target(self, cfg, joints_3d, joints_3d_visible,
+                                kernel):
+        """Generate the target heatmap via "Megvii" approach.
+
+        Args:
+            cfg (dict): data config
+            joints_3d: np.ndarray ([num_joints, 3])
+            joints_3d_visible: np.ndarray ([num_joints, 3])
+            kernel: Kernel of heatmap gaussian
+        Returns:
+            tuple: A tuple containing targets.
+
+            - target: Target heatmaps.
+            - target_weight: (1: visible, 0: invisible)
+        """
+
+        num_joints = cfg['num_joints']
+        image_size = cfg['image_size']
+        heatmap_size = cfg['heatmap_size']
+        heatmaps = np.zeros((num_joints, heatmap_size[1], heatmap_size[0]),
+                            dtype='float32')
+        target_weight = np.zeros((num_joints, 1), dtype=np.float32)
+
+        for i in range(num_joints):
+            heatmap_vis = joints_3d_visible[i, 0]
```

Comment:
`heatmap_vis` is not used very often in this function

Reply:
OK. I'll use target_weight[i] = joints_3d_visible[i, 0] instead.